test/MUXControlsReleaseTest/LibraryThatUsesMUX/LibraryThatUsesMUX.csproj

```diff
@@ -134,7 +134,7 @@
       <Version>6.2.8</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.1.190606001</Version>
+      <Version>2.5.0</Version>
```

Comment:
Is this intentional? I think this will break the "ReleaseTest" workflow since it searches for a version string to replace.

Reply:
Yes it was intentional. The reason being that because the nupkg now has a dependency on WebView2 and I was finding that the nupkg wasn't getting restored in building this solution. I don't know if that was expected but I chose to resolve it by locking to a stable public version that didn't depend on WebView2. 

I can add a comment or I can try to figure out what doesn't work if I let this be updated as part of the pipeline.

Reply:
In that case I think you want to update test\MUXControlsReleaseTest\updateUsedNugetPackageVersion.ps1

Reply:
Since as things stand, the "Release Tests" will always run against 2.5.0 instead of against the newly built package from the Pipeline.

Reply:
It also seems like you missed a few instances of "2.1.190606001" in some of the other projects. It should be consistent.

Reply:
So my intention was that these libraries compile against 2.5 and then *run* against the latest thing. This is similar to how you might expect a middleware library to work. They compile against some version of WinUI but then the consuming app may be choosing a newer version.

Reply:
Ok, I see.

I have two comments on this:

1. Have we decided to support this scenario? It is certainly valuable. But I don't think we currently have processes in place to enable it. This scenario would require us to guarantee ABI compatibility across releases (like we do with Windows). This is a significant amount of work and I don't think we are currently set up to do it. If this is not something that we are supporting, we should not have tests covering it.

2. If we do decide to support this scenario, it is still a *different* scenario from what we are currently testing with these projects. As such it should not replace what we currently have, instead it should be new coverage.

Reply:
Fair enough. I'm trying to put it back and resolve it differently.

Reply:
I fixed this by putting a WebView2 nupkg reference directly from this library. In particular packages.config doesn't support new dependencies showing up in the way we're doing package restore so that's why we need the dependency explicitly. If we ever switch this to PackageReference we could get rid of it, i think.

Reply:
Thanks. That makes sense.